.github/scripts/test_java_demo.py

```diff
@@ -60,24 +75,32 @@ def main():
     for params in PARAMS:
         task = params['task']
         configs = params['configs']
-        java_demo_cmd = [
-            'java', '-cp', 'csrc/mmdeploy/apis/java:demo/java',
-            'demo/java/' + task + '.java', 'cpu'
-        ]
+        input_type = params['input_type']
+        java_command = '\"cpu'
         for config in configs:
             model_url = config
             os.system('wget {} && tar xvf {}'.format(model_url,
                                                      model_url.split('/')[-1]))
             model_dir = model_url.split('/')[-1].split('.')[0]
-            java_demo_cmd.append(model_dir)
-        java_demo_cmd.append('/home/runner/work/mmdeploy/mmdeploy/demo' +
-                             '/resources/human-pose.jpg')
-        java_demo_cmd_str = ' '.join(java_demo_cmd)
-        os.system('export JAVA_HOME=/home/runner/work/mmdeploy/mmdeploy/' +
-                  'jdk-18 && export PATH=${JAVA_HOME}/bin:${PATH} && java' +
-                  ' --version && export LD_LIBRARY_PATH=/home/runner/work/' +
-                  'mmdeploy/mmdeploy/build/lib:${LD_LIBRARY_PATH} && ' +
-                  java_demo_cmd_str)
+            java_command += (' ' + model_dir)
+        if input_type == 'image':
```

Comment:
Is `input_type` necessary? I think 'task' is enough to determine the input source.